templates/download/desktop/index.html

```diff
@@ -127,10 +127,142 @@ <h2>Ubuntu {{ releases.lts.full_version }} LTS</h2>
           </ul>
         </div>
         <!-- tab 2 -->
-        <div class="p-tabs__content"
-             id="system-requirements"
+        <div id="system-requirements-{{ releases.lts.slug }}"
+             class="p-tabs__content"
+             role="tabpanel"
+             aria-labelledby="system-requirements-tab-{{ releases.lts.slug }}">
+          <ul class="p-list--divided">
+            <li class="p-list__item is-ticked">2 GHz dual-core processor or better</li>
+            <li class="p-list__item is-ticked">4 GB system memory</li>
+            <li class="p-list__item is-ticked">25 GB of free hard drive space</li>
+            <li class="p-list__item is-ticked">Either a USB port or a DVD drive for the installer media</li>
+            <li class="p-list__item is-ticked">Internet access is helpful</li>
+          </ul>
+        </div>
+        <!-- tab 3 -->
+        <div id="how-to-install-{{ releases.lts.slug }}"
+             class="p-tabs__content"
+             role="tabpanel"
+             aria-labelledby="how-to-install-tab-{{ releases.lts.slug }}">
+          <p>To install or try Ubuntu Desktop:</p>
+          <ol class="p-list--divided">
+            <li class="p-list__item">Download the ISO image</li>
+            <li class="p-list__item">
+              Create a bootable USB flash drive with <a href="https://etcher.balena.io/">balenaEtcher</a> or similar
+            </li>
+            <li class="p-list__item">Boot your laptop or PC from the USB flash drive</li>
+          </ol>
+          <hr class="p-rule" />
+          <p>
+            <a href="/tutorials/install-ubuntu-desktop">Follow the step-by-step tutorial&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <div class="p-section">
+          <h2>Ubuntu {{ releases.latest.full_version }}</h2>
+        </div>
+        <div class="p-image-wrapper u-hide--small u-hide--medium">
+          {{ image(url="https://assets.ubuntu.com/v1/91be1162-Oracular%20Oriole%20Mascot%20transparent%20background.png",
+          alt="Oracular Oriole",
+          width="200",
+          height="200",
+          hi_def=True,
+          loading="auto") | safe
+          }}
+        </div>
+      </div>
+      <div class="col">
+        <div class="p-section--shallow">
+          <p>
+            The latest version of the Ubuntu operating system for desktop PCs and laptops, Ubuntu 24.10 comes with nine months of security and maintenance updates, until July 2025.
+          </p>
+          <hr />
```

Comment:
Rules which are not dividing sections should be muted
```suggestion
          <hr class="p-rule--muted"/>
```